src/features/PopularMovies/MovieDetails/Header/index.js

```diff
@@ -10,12 +12,19 @@ import {
     MovieDetailsHeader
 } from "./styled";
 
-
 export const Header = () => {
     const movieData = useSelector(selectMovieData);
+    const configuration = useSelector(selectConfiguration);
+    const screenWidth = useGetScreenWidth();
+
+    let size;
+
+    if (screenWidth > 1280) size = configuration.images.backdrop_sizes[3];
+    if (screenWidth > 780 && screenWidth < 1281) size = configuration.images.backdrop_sizes[2];
+    if (screenWidth > 300 && screenWidth < 781) size = configuration.images.backdrop_sizes[1];
+    if (screenWidth < 301) size = configuration.images.backdrop_sizes[0];
```

Comment:
It's a bit confusing. Witch picture are we gonna get on 1280.5px screen width or 780.5px or 300.5px?
Shouldn't  be something like this or similar?
```
if (screenWidth >= 1280) size = configuration.images.backdrop_sizes[3];
if (screenWidth >= 780 && screenWidth < 1280) size = configuration.images.backdrop_sizes[2];
if (screenWidth >= 300 && screenWidth < 780) size = configuration.images.backdrop_sizes[1];
if (screenWidth < 300) size = configuration.images.backdrop_sizes[0];
```

Reply:
should be:
>1280
>780 &&  <=1280 etc